spec/models/note_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Note, type: :model do
+  subject(:note) do
+    create(:note)
+  end
```

Comment:
Para este caso con hacer un `build(:note)` alcanza y tambien podes usar la sintaxis de llaves para ahorrarte unas lineas:
```suggestion
  subject(:note) { build(:note) }
```

Reply:
@JoaquinSeita intenté probar con build y me fallaban los test debido a la relación que tiene con User. Por esta razón mantuve el "create". Lo revisé con Flor y me explicó que se debe a que build realmente no está creando el registro en la DB y por eso no genera la asociación con User.